### ISSUE TITLE
sst-info reports redundant partitioners when SST-Core is built

### ISSUE THREAD

**Issue by allevin** [sstsimulator/sst-core#323]
After building sst-core but not sst-elements (with zoltan included), I ran sst-info.  I get redundant partitioner entries for linear, roundrobin, simple and zoltan.   As follows:

```PROCESSED 1 .so (SST ELEMENT) FILES FOUND IN DIRECTORY(s) 
================================================================================
ELEMENT 0 = sst (SST Core elements)
   NUM COMPONENTS    = 0
   NUM EVENTS        = 0
   NUM MODULES       = 0
   NUM SUBCOMPONENTS = 0
   NUM PARTITIONERS  = 10
      PARTITIONER 0 = linear (Partitions components by dividing Component ID space into roughly equal portions.  Components with sequential IDs will be placed close together.)
      PARTITIONER 1 = roundrobin (Partitions components using a simple round robin scheme based on ComponentID.  Sequential IDs will be placed on different ranks.)
      PARTITIONER 2 = simple (Simple partitioning scheme which attempts to partition on high latency links while balancing number of components per rank.)
      PARTITIONER 3 = self (Used when partitioning is already specified in the configuration.)
      PARTITIONER 4 = zoltan (Zoltan parallel partitioner)
      PARTITIONER 5 = linear (Partitions components by dividing Component ID space into roughly equal portions.  Components with sequential IDs will be placed close together.)
      PARTITIONER 6 = roundrobin (Partitions components using a simple round robin scheme based on ComponentID.  Sequential IDs will be placed on different ranks.)
      PARTITIONER 7 = simple (Simple partitioning scheme which attempts to partition on high latency links while balancing number of components per rank.)
      PARTITIONER 8 = single (Allocates all components to rank 0.  Automatically selected for serial jobs.)
      PARTITIONER 9 = zoltan (zoltan parallel partitioner)
   NUM GENERATORS    = 0
```
What is interesting is that the "self" partitioner is not duplicated.  This may be related to #322

**Comment by jpvandy** [comment]
note, that single also appears only once

**Comment by feldergast** [comment]
Fixed in 7.2.
